content/collections/advanced-techniques/en/advanced-metric-use-cases.md

```diff
@@ -36,7 +36,7 @@ For example, a funnel analysis includes all users who meet its filter requiremen
 
 ### Analyze your experiment data using other Amplitude Analytics metrics
 
-Amplitude Analytics offers metrics that Amplitude Experiment doesn't. You can also use the steps in the previous section to analyze things like time to convert or [retention](/docs/analytics/charts/retention-analysis/retention-analysis-build) (though these measurements only consider a user's first conversion). 
+Amplitude Analytics offers metrics that Amplitude Experiment doesn't. You can also use the steps in the previous section to analyze things like time to convert or [return on or after retention](/docs/analytics/charts/retention-analysis/retention-analysis-build) (though these measurements only consider a user's first conversion). 
```

Comment:
we support return on retention in experiment but we dont support return on or after retention in experiment